src/ReactiveDomain/ReactiveDomain/FrameFormats/TwoByte128X128.cs

```diff
@@ -9,10 +9,7 @@ namespace ReactiveDomain.FrameFormats
     [StructLayout(LayoutKind.Sequential)]
     public unsafe struct TwoByte128X128Frame
     {
-        public long FrameNumber;
-        public fixed byte FrameId[16];
-        public fixed byte VideoId[16];
-        public double OffsetMilliseconds;
+        public VideoFrameHeader FrameHeader;
```

Comment:
there is an expectation that the structs have the same layout in memory, this doesn't seem to follow that pattern.